Library/Homebrew/shims/linux/super/cc

```diff
@@ -311,6 +311,10 @@ class Cmd
     paths.map! { |path| prefix + path }
   end
 
+  def rpath_flags(prefix, paths)
+    paths.uniq.map { |path| prefix + path }
```

Comment:
Note that non-exsting `lib` directories of dependencies are already pruned from this list earlier: https://github.com/Linuxbrew/brew/blob/master/Library/Homebrew/extend/ENV/super.rb#L193

Pruning again here was a bit redundant.